docs/install/from_source.rst

```diff
@@ -293,6 +303,17 @@ like ``virtualenv``.
 
        pip3 install --user tornado psutil xgboost cloudpickle
 
+Note on M1 macs, you may have trouble installing xgboost / scipy. A workaround for this is to do the following commands:
```

Comment:
I like to explain why there might be an issue. For example, it looks like OPENBLAS needs to be installed and configured. Stating that helps future readers and contributors understand what the problem is, and possibly update the instructions if the issue is resolved. So, for example:

"`scipy` and `xgboost` requires some additional dependencies to be installed, including `openblas` and its dependencies. Use the following commands to install `scipy` and `xgboost` with the required dependencies and configuration."

Reply:
Done